src/components/Forms/FormAgreements.vue

```diff
@@ -114,4 +114,13 @@ export default {
 <style scoped>
 @import '../../../assets/global-classes.less';
 @import '../../../assets/color-constants.less';
+
+label > .input-primary {

```

Comment:
Do we actually want this styling? What does this do to this page? I would assume the checkbox labels are different since they're to the side of the inputs

Reply:
It makes the labels appear right on top of the inputs instead of a bit above. Let me know if you want to keep it or not.

Reply:
There is nothing with the class "input-primary" on this page so this styling does nothing. The block below this is also definitely not what we want because the checkbox text is in a paragraph so 10px is way too small. We're probably better off without the pointer cursor as well. The whole point of not just throwing things into global-classes is so that we can control any side effects styling has but that's worthless if we're just going to throw blocks of styling on components without checking them

Reply:
My bad, I put this styling on the wrong page. Should be good now.